frontend/src/container/ListOfDashboard/TableComponents/DeleteButton.tsx

```diff
@@ -20,13 +24,14 @@ function DeleteButton({ deleteDashboard, id }: DeleteButtonProps): JSX.Element {
 			onOk() {
 				deleteDashboard({
 					uuid: id,
+					refetch: refetchDashboardList,
 				});
 			},
 			okText: 'Delete',
 			okButtonProps: { danger: true },
 			centered: true,
 		});
-	}, [id, modal, deleteDashboard]);
+	}, [modal, deleteDashboard, id, refetchDashboardList]);
```

Comment:
`id` isn't a good variable name by itself.May be, `dashboardId` ?

Reply:
good suggestion @Damans227 though this is not related to this PR